airbyte-server/src/main/java/io/airbyte/server/handlers/WebBackendConnectionsHandler.java

```diff
@@ -388,7 +388,6 @@ public WebBackendConnectionRead webBackendUpdateConnectionNew(final WebBackendCo
     final ConfiguredAirbyteCatalog existingConfiguredCatalog =
         configRepository.getConfiguredCatalogForConnection(connectionId);
     ConnectionRead connectionRead;
-    connectionRead = connectionsHandler.updateConnection(connectionUpdate);
```

Comment:
I don't think this change is necessary - since we are capturing the existing configured catalog before we update the connection, it is fine to execute this call here

Reply:
reverted, I was wondering why the test were not passing it was not the root cause